### DISCUSSION TITLE
difference in fragment ion intensity pattern for the same peptide in same sample

### DISCUSSION THREAD

**Discussion by amrelguo** [vdemichev/DiaNN#1310] [Q&A]
Hi Vadim,
When I searched the DIA data for the same sample twice, considering the same parameter settings and a different DDA library, they showed completely different intensity patterns for their fragment ion data for some peptides in the same sample. However, both DDA libraries have almost the same search space, and the intensity pattern of the corresponding peptide in the DDA library is exactly the same.
![Picture1](https://github.com/user-attachments/assets/91b15ef9-462f-4226-b55c-0e1c552c786e)
![Picture2](https://github.com/user-attachments/assets/b3344f68-c47f-48dd-87fc-71f234fbb259)

**Comment by vdemichev** [comment]
Hi,

This is likely a match to an incorrect peptidoform. Can happen randomly, to control can use peptidoform q-values in DIA-NN 1.9.2.

Best,
Vadim
